index.html

```diff
@@ -4580,6 +4580,319 @@ <h1>Current Issues</h1>
+    the types of a relationship (e.g., "parent" or "employee"), or numeric
+    values (e.g., zero, one, and infinity).
+</blockquote>
+  <p>
+  In other words, it does not matter whether a resource is “on” or “off” the
+  Internet—if it can be identified, it can be assigned a URI, and therefore it
+  can be assigned a <a>DID</a>.
+  </p>
+  <h2>How do you know what a DID identifies?</h2>
+  <p>
+  For any <a>DID</a>, the <a>DID controller</a> determines the <a>DID subject</a>.
+  Unfortunately it is all but impossible to determine the <a>DID subject</a>
+  from looking at the <a>DID</a> itself. The reason is that, in order to satisfy
+  several core properties of a <a>DID</a> as an identifier—especially
+  decentralization and cryptographic verifiability—<a>DIDs</a> are generally
+  only meaningful to machines, not humans. To illustrate, compare the following
+  two URIs:
+  </p>
+    <p>
+       <code>https://www.w3.org/2019/did-wg/WorkMode/getting-started</code>
+    </p>
+    <p>
+        <code>did:example:8uQhQMGzWxR8vw5P3UWH1j</code>
+    </p>
+    <p>
+    The first is the URL of the Getting Started page of the W3C DID Working
+    Group. This is a human-meaningful identifier (at least to someone who
+    understands the English language). In this sense, the reader can be said to
+    “know” what the URL identifies without having to dereference it (provided
+    the reader trusts the publisher’s label).
+    </p>
+  <p>
+  The second URI—the example <a>DID</a>—is meaningless to humans no matter what
+  language you speak. What it identifies is anyone’s guess in the absence of
+  further information describing the <a>DID subject</a>. So further information
+  about the <a>DID subject</a> is only discoverable by resolving the <a>DID</a>
+  to the <a>DID document</a>, obtaining a verifiable credential about the
+  <a>DID</a>, or via some other description of the <a>DID</a>.
+  </p>
+    <h2>Does the DID identify the DID document?</h2>
+    <p>
+    No. To be very precise, the <a>DID</a> identifies the <a>DID subject</a> and
+    <em>resolves to</em> the <a>DID document</a> (by following the protocol
+    specified by the <a>DID method</a>). The <a>DID document</a> is not a
+    separate resource from the <a>DID subject</a> and does not have a URI
+    separate from the <a>DID</a>. Rather the <a>DID document</a> is an artifact
+    of <a>DID resolution</a> controlled by the <a>DID controller</a> for the
+    purpose of describing the <a>DID subject</a>.
+    </p>
+    <p>
+    This distinction is illustrated by the graph model shown in figure 2.
+    </p>
+        <figure id="did-and-did-document-graph">
+          <img style="margin: auto; display: block; width: 75%;"
+            src="diagrams/figure-a.1-did-and-did-document-graph.png" alt="
+Diagram showing a graph model for how DID controllers assign DIDs to identify
+DID subjects and resolve to DID documents that describe the DID subjects.
```

Comment:
I totally agree, @msporny. One question: do I need to do this full description for each of the figures in the appendix? And it may take quite a few words to describe the diagram—is that okay?

Reply:
Yes, we need to do this as a part of our accessibility requirements: https://github.com/w3c/did-core/issues/118 -- the description should go in a `longdesc` attribute associated with each image so that people using screen readers will be able to hear the descriptive text while those that can rely on their sense of sight will be able to look at the image.